docs/development/i18n.rst

```diff
@@ -64,6 +59,19 @@ files into the corresponding ``*.j2`` file and committed to the
 SecureDrop repository. They are then installed when configuring Tails
 with the ``tasks/create_desktop_shortcuts.yml`` tasks.
 
+We don't expect translators to deal with all these files directly, of
+course. Translation happens on our `Weblate`_ server, which is
+configured to use a fork of the main `SecureDrop git repository`_.
+
+At the start of the the release process, the localization manager
+collects string changes on the ``develop`` branch and merges them to
+the ``i18n`` branch of the ``securedrop-i18n`` repository. The changes
+will then appear in Weblate, and translation can begin. Once complete,
+the localization manager collects the changes to the PO files on
+``securedrop-i18n/i18n`` in a pull request for ``develop``. Once that
```

Comment:
a link to securedrop-i18n would be nice. you could also mention that you need to pull the latest from `securedrop/develop` to `securedrop-i18n/develop` before making a pr to merge to `securedrop-i18n/i18n`. i also think you should stick to this format of menioning repo + "/" + branch name because it adds clarity.